testing/fil_token_integration/tests/common.rs

```diff
@@ -0,0 +1,163 @@
+
+    Tester::new(NetworkVersion::V15, StateTreeVersion::V4, bundle_root, blockstore.clone()).unwrap()
+}
+
+/// Helper routines to simplify common operations with a Tester
+pub trait TestHelpers {
+    /// Call a method on an actor
+    fn call_method(
+        &mut self,
+        from: Address,
+        to: Address,
+        method_num: u64,
+        params: Option<RawBytes>,
+    ) -> ApplyRet;
+
+    /// Install an actor with initial state and ID
+    /// Returns the actor's address
+    fn install_actor_with_state<S: Serialize>(
+        &mut self,
+        path: &str,
+        actor_id: u64,
+        state: S,
+    ) -> Address;
+
+    /// Install an actor with no initial state
+    /// Takes ID and returns the new actor's address
+    fn install_actor_stateless(&mut self, path: &str, actor_id: u64) -> Address;
+}
+
+/// Helper routines to simplify common token operations
+pub trait TokenHelpers {
+    /// Get balance from token actor for a given address
+    /// This is a very common thing to check during tests
+    fn get_balance(
+        &mut self,
+        operator: Address,
+        token_actor: Address,
+        target: Address,
+    ) -> TokenAmount;
+
+    fn mint_tokens(
+        &mut self,
+        operator: Address,
+        token_actor: Address,
+        target: Address,
+        amount: TokenAmount,
+        operator_data: RawBytes,
+    ) -> ApplyRet;
+}
+
+impl<B: Blockstore, E: Externs> TestHelpers for Tester<B, E> {
+    fn call_method(
+        &mut self,
+        from: Address,
+        to: Address,
+        method_num: u64,
+        params: Option<RawBytes>,
+    ) -> ApplyRet {
+        static mut SEQUENCE: u64 = 0u64;
+        let message = Message {
+            from,
+            to,
+            gas_limit: 99999999,
+            method_num,
+            sequence: unsafe { SEQUENCE },
+            params: if let Some(params) = params { params } else { RawBytes::default() },
+            ..Message::default()
+        };
+        unsafe {
+            SEQUENCE += 1;
+        }
+        self.executor.as_mut().unwrap().execute_message(message, ApplyKind::Explicit, 100).unwrap()
+    }
+
+    fn install_actor_with_state<S: Serialize>(
+        &mut self,
+        path: &str,
+        actor_id: u64,
+        state: S,
+    ) -> Address {
+        let code = load_actor_wasm(path);
+        let address = Address::new_id(actor_id);
+        let state_cid = self.set_state(&state).unwrap();
+        self.set_actor_from_bin(&code, state_cid, address, TokenAmount::zero()).unwrap();
+        address
+    }
+
+    fn install_actor_stateless(&mut self, path: &str, actor_id: u64) -> Address {
+        let code = load_actor_wasm(path);
+        let address = Address::new_id(actor_id);
+        self.set_actor_from_bin(&code, Cid::default(), address, TokenAmount::zero()).unwrap();
+        address
+    }
+}
+
+impl<B: Blockstore, E: Externs> TokenHelpers for Tester<B, E> {
+    fn get_balance(
```

Comment:
Call it token balance to make slightly less confusing with the native FIL token balance

Reply:
Will do. I'm also thinking of splitting it off into another module later - we'd have `common.rs` with the basic helpers for calling methods and installing actors but then separate FRC46 and NFT modules for things like minting, balance checks, etc.